### ISSUE TITLE
[new IA] IA-CAESAR

### ISSUE THREAD

**Issue by myrosblog** [Qwant/Instant-Answers#31]
Hi,

I'w working on an Caesar IA, with the following specs :
```
caesar Qwant = "Djnag" (default offset is 13)

caesar 10 Qwant = "Agkxd" (offset set to 10)

caesar 158 Qwant is great ! = "Sycpv ku itgcv !" (offset set to 2, no manipulation on non-letter character)
```

<img width="220" alt="demo caesar Qwant" src="https://user-images.githubusercontent.com/9380369/31586644-f5a45e3e-b1d3-11e7-98f7-5fb4572e6f41.png">
<img width="200" alt="demo caesar 10 Qwant" src="https://user-images.githubusercontent.com/9380369/31586645-f5bfe1d6-b1d3-11e7-9589-313803b8150c.png">
<img width="200" alt="demo caesar 158 Qwant is great !" src="https://user-images.githubusercontent.com/9380369/31586675-5dfb7cf6-b1d4-11e7-8f9d-9cd417c0835d.png">

I will certainly add a 'bruteforce' mode (displaying all 26 possibilities), either by doing `caesar bruteforce` or with a frontend JS script

Are you interested ?
Thanks

**Comment by alexisvigoureux** [comment]
Hello Florian, we are very interested in your Caesar IA. But your IA should be added as an option to the encryption IA rather than a brand new IA: https://github.com/Qwant/Instant-Answers/blob/master/src/modules/encryption/encryption.js
Our wiki talks about that: https://github.com/Qwant/Instant-Answers/wiki/5.3.-Tutorial-3:-Enhancing-an-existing-IA-(Encryption)
Make a Pull Request and our team will review your code to add it to Qwant

**Comment by myrosblog** [comment]
closed with #32
